package.json

```diff
@@ -46,6 +46,7 @@
     "core-js": "2.5.7",
     "hammerjs": "2.0.8",
     "lodash": "4.17.10",
+    "particles.js": "^2.0.0",
```

Comment:
This is not required as you included the script via a cdn in `index.html` (which is a better way). Remove this change (file) from your commit.